info/v1/container.go

```diff
@@ -432,6 +448,15 @@ type UdpStat struct {
 	TxQueued uint64
 }
 
+func (u *UdpStat) UdpStatMap() map[string]uint64 {
```

Comment:
I would prefer not to add functions here.  Do you think you could just create the list within the prometheus function like `container_tasks_state`?  Same goes for the TcpStatMap function